etl/steps/data/garden/emissions/2023-11-23/national_contributions.meta.yml

```diff
@@ -1,330 +1,323 @@
         short_unit: °C
-        description: |
-          Change in global mean surface temperature (in °C) caused by greenhouse gas emissions from agriculture and land use.
+        description_short: *measured-in-celsius
       temperature_response_ghg_total:
         title: Change in global mean surface temperature caused by greenhouse gas emissions
         unit: °C
         short_unit: °C
-        description: |
-          Change in global mean surface temperature (in °C) caused by greenhouse gas emissions. This measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of carbon dioxide, methane and nitrous oxide. The warming effects of each gas are calculated based on cumulative CO2-equivalent emissions using the Global Warming Potential (GWP*) approach.
+        description_short: *measured-in-celsius
+        description_key:
+          - This temperature change measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of carbon dioxide, methane and nitrous oxide.
+          - The warming effects of each gas are calculated based on cumulative CO₂-equivalent emissions using the Global Warming Potential (GWP*) approach.
       temperature_response_ch4_fossil:
         title: Change in global mean surface temperature caused by methane emissions from fossil fuels and industry
         unit: °C
         short_unit: °C
-        description: |
-          Change in global mean surface temperature (in °C) caused by methane emissions from fossil fuels and industry.
+        description_short: *measured-in-celsius
       temperature_response_ch4_land:
         title: Change in global mean surface temperature caused by methane emissions from agriculture and land use
         unit: °C
         short_unit: °C
-        description: |
-          Change in global mean surface temperature (in °C) caused by methane emissions from agriculture and land use.
+        description_short: *measured-in-celsius
       temperature_response_ch4_total:
         title: Change in global mean surface temperature caused by methane emissions
         unit: °C
         short_unit: °C
-        description: |
-          Change in global mean surface temperature (in °C) caused by methane emissions. This measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of methane. The warming effects of each gas are calculated based on cumulative CO2-equivalent emissions using the Global Warming Potential (GWP*) approach.
+        description_short: *measured-in-celsius
+        description_key:
+          - This temperature change measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of methane.
+          - The warming effects of each gas are calculated based on cumulative CO₂-equivalent emissions using the Global Warming Potential (GWP*) approach.
```

Comment:
again, maybe add another bullet point for clarifying what Global Warming Potential (GWP*) approach is?